### DISCUSSION TITLE
Is there a clipping extension for Chrome browser?

### DISCUSSION THREAD

**Discussion by tankb52** [zim-desktop-wiki/zim-desktop-wiki#2584] [Q&A]
I know there's a zim-clip extension on Firefox.
Is there a similar extension on the Chrome browser?

**Comment by Nick222** [comment]
@rui-nibau  ?

**Comment by rui-nibau** [comment]
Hello.

Never try to install zim-clip on Chrome. Don't know if it works on it or not.

BTW, zim-clip is a little bit old and i have some issues udpating it on amo (addons.mozilla.org) since a couple of years (long story).

I'm working on a new browser addon that will allow the same things than zim-clip and a little more.

Ready, well, when it's ready (first quarter of 2026 i hope)
